engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicPropertyExtractor.scala

```diff
@@ -3,16 +3,19 @@ package pl.touk.nussknacker.engine.management.periodic
 import com.typesafe.scalalogging.LazyLogging
 import pl.touk.nussknacker.engine.api.deployment.{CustomProcess, GraphProcess, ProcessDeploymentData}
 import pl.touk.nussknacker.engine.management.periodic.CronPropertyExtractor.CronPropertyDefaultName
+import pl.touk.nussknacker.engine.management.periodic.PeriodicPropertyExtractor.defaultScheduleName
 import pl.touk.nussknacker.engine.marshall.ProcessMarshaller
 
 import java.time.Clock
 
 trait PeriodicPropertyExtractor {
-  def apply(processDeploymentData: ProcessDeploymentData): Either[String, PeriodicProperty]
+  def apply(processDeploymentData: ProcessDeploymentData): Either[String, Map[String, PeriodicProperty]]
 }
 
 object PeriodicPropertyExtractor {
 
+  val defaultScheduleName = "schedule"
```

Comment:
What's the trick with this default schedule? Will it be gone?

Reply:
Mostly for backward compatibility and for simple cases where we have only one schedule

Reply:
I wonder if we can get rid of backward compatibility, make this multiple schedules approach as easy that it won't be needed maintenance of two schemes of creation of schedules?

Reply:
I ended up having sealed trait... I still think in many cases single schedule would be better...